microsoft/testsuites/mdatp/check-mdatp.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+
+# check the following folders for printable files, print them
+FOUND_DEFENDER=1
+MDATP_OPT_DIR='/etc/opt/microsoft/mdatp/'
+MDATP_EXTENSION_DIR='/var/lib/waagent/Microsoft.Azure.AzureDefenderForServers.MDE.Linux'
```

Comment:
Do you want to add MDATP_EXTENSION_LOG_DIR also?
For ARM = /var/log/azure/Microsoft.Azure.AzureDefenderForServers.MDE.Linux
For ARC = /var/lib/GuestConfig/extension_logs/Microsoft.Azure.AzureDefenderForServers.MDE.Linux-<version>